invisible_cities/io/mcinfo_io.py

```diff
@@ -3,6 +3,9 @@
 import numpy  as np
 import pandas as pd
 
+from .. cities.components import get_run_number
+from .. cities.components import get_mc_info_safe
+
```

Comment:
by design, `components` should only be used in the cities. Those functions belong somewhere else.

Reply:
Not sure what am I suppose to do then; notice that those are pre-existing functions. Should I move them to` mcinfo_io`?

Reply:
Well, as discussed below, you will not need them, you will need `get_mc_info`, which is in reco.tbl_function. I think that function should also be moved from there to mcinfo_io, but maybe that's for another PR.